user-interface/src/main/java/life/qbic/datamanager/views/projects/project/info/ProjectSummaryComponent.java

```diff
@@ -1,253 +1,537 @@
-    principalInvestigator.add(generateContactContainer(project.getPrincipalInvestigator()));
-    entries.add(new Entry("Principal Investigator", "", principalInvestigator));
+  private static Button createButtonWithListener(String label,
+      ComponentEventListener<ClickEvent<Button>> listener) {
+    var button = new Button(label);
+    button.addClickListener(listener);
+    return button;
+  }
 
-    var projectResponsible = new Div();
-    project.getResponsiblePerson()
-        .ifPresentOrElse(person -> projectResponsible.add(generateContactContainer(person)),
-            () -> {
-              Span noProjectResponsibleSelected = new Span("No Project Responsible Selected");
-              noProjectResponsibleSelected.addClassName("tertiary");
-              projectResponsible.add(noProjectResponsibleSelected);
-            });
-    entries.add(new Entry("Project Responsible", "", projectResponsible));
+  private static List<? extends UserScopeStrategy> loadScope(Predicate<ProjectId> hasWriteScope,
+      ProjectId id,
+      Section... sections) {
+    if (hasWriteScope.test(id)) {
+      return loadWriteScope(sections);
+    } else {
+      return loadReadScope(sections);
+    }
+  }
 
-    var projectManager = new Div();
-    projectManager.add(generateContactContainer(project.getProjectManager()));
-    entries.add(new Entry("Project Manager", "", projectManager));
+  private static List<? extends UserScopeStrategy> loadReadScope(Section[] sections) {
+    return Arrays.stream(sections).map(ReadScopeStrategy::new).toList();
+  }
 
-    return entries;
+  private static List<? extends UserScopeStrategy> loadWriteScope(Section[] sections) {
+    return Arrays.stream(sections).map(WriteScopeStrategy::new).toList();
   }
 
-  private static Div generateContactContainer(Contact contact) {
-    Span nameSpan = new Span(contact.fullName());
-    Span emailSpan = new Span(contact.emailAddress());
-    Div personContainer = new Div(nameSpan, emailSpan);
-    personContainer.addClassName("person-contact-display");
-    emailSpan.addClassNames("email");
-    nameSpan.addClassName("name");
-    return personContainer;
+  private String formatDate(Instant date) {
+    var formatter = DateTimeFormatter.ofPattern(DATE_TIME_PATTERN).withZone(ZoneId.systemDefault());
+    return formatter.format(date);
   }
 
   public void setContext(Context context) {
-    if (context.projectId().isEmpty()) {
-      throw new ApplicationException("no project id in context " + context);
+    this.context = Objects.requireNonNull(context);
+    var projectId = context.projectId()
+        .orElseThrow(() -> new ApplicationException("No project id provided"));
+    var projectOverview = projectInformationService.findOverview(projectId)
+        .orElseThrow(() -> new ApplicationException("No project with given ID found"));
+    var fullProject = projectInformationService.find(projectId)
+        .orElseThrow(() -> new ApplicationException("No project found"));
```

Comment:
Since we don't differentiate between individual reloads, you could also call the reloadInformation() method here which will rebuild all the information.